src/wolfio.c

```diff
@@ -121,6 +124,12 @@ static WC_INLINE int wolfSSL_LastError(int err)
 
 static int TranslateIoError(int err)
 {
+#ifdef  _WIN32
+    size_t errstr_offset;
```

Comment:
```[check-source-text] [2 of 2] [ba42ebca76]
weird control chars, hard tabs, CRs, trailing whitespace:
./src/wolfio.c:127:#ifdef  _WIN32©
./src/wolfio.c:129:    char errstr[WOLFSSL_STRERROR_BUFFER_SIZE];©
C++-style comments:
./src/wolfio.c:130:#endif //  _WIN32
    check-source-text fail_BC```

Reply:
hopefully that is an any not all as I see the CR but none of the other 3.

![image](https://github.com/wolfSSL/wolfssl/assets/1643324/bc3a68d7-6e0a-4d09-9b9b-fe57469fb9a7)

the merged #6534 should hopefully auto-handle these things in the future.